WeakAuras/Prototypes.lua

```diff
@@ -10113,6 +10113,210 @@ Private.event_prototypes = {
+        test = "true",
+      },
+      {
+        name = "total",
+        init = "currencyInfo.maxQuantity",
+        hidden = true,
+        store = true,
+        test = "true",
+      },
+      {
+        name = "progressType",
+        init = "'static'",
+        hidden = true,
+        store = true,
+        test = "true",
+      },
+      {
+        name = "icon",
+        init = "currencyInfo.iconFileID",
+        store = true,
+        hidden = true,
+        test = "true",
+      },
+      {
+        name = "maxQuantity",
+        init = "currencyInfo.maxQuantity",
+        type = "number",
+        display = L["Maximum Quantity"],
+        store = true,
+        conditionType = "number",
+      },
+      {
+        name = "capped",
+        init = [[quantity > 0 and quantity >= maxQuantity]],
+        type = "toggle",
+        display = L["Capped"],
+        store = true,
+        conditionType = "bool",
+      },
+      {
+        name = "canEarnPerWeek",
+        init = "currencyInfo.canEarnPerWeek",
+        type = "toggle",
+        display = L["Has weekly maximum"],
+        store = true,
+        conditionType = "bool",
+      },
+      {
+        name = "quantityEarnedThisWeek",
+        init = "currencyInfo.quantityEarnedThisWeek",
+        type = "number",
+        display = L["Quantity earned this week"],
+        store = true,
+        conditionType = "number",
+        enable = function(trigger) return trigger.use_canEarnPerWeek end,
+        test = "true",
+      },
+      {
+        name = "maxWeeklyQuantity",
+        init = "currencyInfo.maxWeeklyQuantity",
+        store = true,
+        hidden = true,
+        test = "true",
+      },
+      {
+        name = "weeklyCapped",
+        init = [[quantityEarnedThisWeek >= maxWeeklyQuantity]],
+        type = "toggle",
+        display = L["Capped this week"],
+        store = true,
+        conditionType = "bool",
+        enable = function(trigger) return trigger.use_canEarnPerWeek end,
+        test = "true",
+      },
+      {
+        name = "description",
+        init = "currencyInfo.description",
+        type = "string",
+        display = L["Description"],
+        store = true,
+        hidden = true,
+        test = "true",
+        conditionType = "string",
+      },
+      {
+        name = "isHeader",
+        init = "currencyInfo.isHeader",
+        type = "toggle",
+        display = L["Is Header"],
+        store = true,
+        hidden = true,
+        test = "true",
+        conditionType = "bool",
+      },
+      {
+        name = "isTypeUnused",
+        init = "currencyInfo.isTypeUnused",
+        type = "toggle",
+        display = L["Is Type Unused"],
+        store = true,
+        hidden = true,
+        test = "true",
+        conditionType = "bool",
+      },
+      {
+        name = "isShowInBackpack",
+        init = "currencyInfo.isShowInBackpack",
+        type = "toggle",
+        display = L["Is Show in Backpack"],
```

Comment:
It was a typo/copypaste from blizzards field name, however since this isn't even supported in-game (the state of the checkbox in the game UI isn't represented in the data given by the API) I've removed it together with isTypeUnused. https://github.com/WeakAuras/WeakAuras2/pull/4672/commits/b152be067e14d097d16f6a9a928507e31405c55d